editors/intellij/src/main/resources/messages/BiomeBundle.properties

```diff
@@ -1,5 +1,6 @@
 name=Biome
 biome.path.label=Biome CLI Path
+biome.config.path.label=biome.json Directory Path
```

Comment:
```suggestion
biome.config.path.label=Configuration directory path
```

Reply:
I'm fine to make that change, although I do wonder if `Configuration` is slightly less clear than the `biome.json`. Also, if we are making the casing lowercase, should we make the `biome.path.label` above `Biome CLI path`?

Reply:
Yeah it makes !